src/main/webapp/resources/js/pages/projects/linelist/components/Toolbar/Toolbar.jsx

```diff
@@ -85,10 +86,14 @@ export class Toolbar extends Component {
               />
             </Form.Item>
             <Form.Item>
-              <LineListTour
-                isOpen={this.state.tourOpen}
-                closeTour={this.closeTour}
-              />
+              <Suspense fallback={<span />}>
+                {this.state.tourOpen ? (
+                  <LineListTour
+                    isOpen={this.state.tourOpen}
+                    closeTour={this.closeTour}
+                  />
+                ) : null}
```

Comment:
Should we not display something to the user such as "loading", etc if the component has not been loaded yet rather than returning null?

Reply:
So normally I would say yes, but in this case no.  The reason is that everything that is needed is on the page.  The downloaded code opens a modal so nothing should be displayed until that is loaded.